packages/theme/components/Checkout/VsfShippingProvider.vue

```diff
@@ -39,6 +50,7 @@ export default {
     const { $spree } = useVSFContext();
     const { setCart } = useCart();
     const { state: shipments, save: saveShipments, load: loadShipments } = useShippingProvider();
+    const getBackToShippingDetails = () => location.reload();
```

Comment:
Hehe, is there a way to do that without reloading the page? I know it's a bit of state management, but would feel much smoother